include/internal/catch_tostring.h

```diff
@@ -152,11 +164,26 @@ namespace Catch {
     struct StringMaker<std::string> {
         static std::string convert(const std::string& str);
     };
+
+#ifdef CATCH_CONFIG_STRING_VIEW
```

Comment:
I would move one of the two specializations so that there is less conditionally compiled blocks of code and things are easier to read, but that is a minor nitpick.

(This also applies to the implementation in the .cpp file)

Reply:
The `std::wstring` specialization should only be present when `CATCH_CONFIG_WCHAR` is defined,
`std::string_view` only when string_view support is on, `std::wstring_view` only when both are on - I could move blocks around so the ifdefs don't nest, but wouldn't it still take three ifdef'ed blocks?